crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -811,6 +811,7 @@ en:
     not_found: 'Not found'
     chef_server_down: 'Chef server is not available'
     param_missing: 'A required parameter is missing'
+    pre_install: 'Crowbar is not installed yet, therefore the API is restricted'
```

Comment:
Maybe change "pre_install" to "before_install" (to avoid potential confusion with preinstalling something).

Also I'd change the string to: "API is not available before installation" (or another simple sentence, as it fits better an error message than a full sentence like yours).